src/main/scala/dpla/ingestion3/mappers/utils/Extractor.scala

```diff
@@ -64,6 +65,27 @@ trait XmlExtractor extends Extractor[NodeSeq] {
       case _ => Seq()
     }
   }
+
+  /**
+    * Exclude nodes that do not have an attribute that matches att and value parameters
+    *
+    * @param node Node XML node to examine
+    * @param att String name of attribute
+    * @param value String value of attribute
+    * @return Boolean
+    */
+  def filterAttribute(node: Node, att: String, value: String): Boolean = (node \ ("@" + att)).text.toLowerCase == value
+
+  /**
+    * Get Nodes that match the given attribute name and attribute value
+    * @param e Elem to examine
+    * @param att String name of attribute
+    * @param value String value of attribute
+    * @return NodeSeq of matching nodes 
+    */
+  def getByAttribute(e: Elem, att: String, value: String): NodeSeq = {
+    e \\ "_" filter { n => filterAttribute(n, att, value) }
+  }
```

Comment:
Not sure what the testing philosophy is for mappings - is `filterAttribute` and `getByAttribute` something you would want to unit test, or are the individual provider mapping tests sufficient?